doc/explanations/min-utxo-alonzo.rst

```diff
@@ -60,7 +60,7 @@ Recall that
 +--------------------------+-----------------+-----------------+-------------------+------------------+------------------+---------------------------------+
 | size of value            | 11              | 12              | 15                | 16               | 17               | 173                             |
 +--------------------------+-----------------+-----------------+-------------------+------------------+------------------+---------------------------------+
-| ``utxoEntrySize``        | 38              | 39              | 42                | 43               | 44               | 20                              |
+| ``utxoEntrySize``        | 38              | 39              | 42                | 43               | 44               | 200                              |
```

Comment:
```suggestion
| ``utxoEntrySize``        | 38              | 39              | 42                | 43               | 44               | 200                             |
```

Reply:
thank you for helping making this better! how did you verify this?

Reply:
Well, I did all the math, but you can verify that 200 multiplied by coinsPerUTxOWord (34,482) gives minUTxO (6,896,400).

Reply:
ah, of course, I forgot we had the 6,896,400 value, thanks!